tasks/update_pr_stats.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+begin
+  require 'net/http'
+  require 'safe_yaml'
+  require_relative './support/github_pr_stats'
+
+  desc 'Update _config.yml stats'
+  task :update_stats do
+    gh_pr_stats = GithubPRStats.new
+    result = gh_pr_stats.update
+
+    next unless ENV['CI'] && ENV['CI']
+    next unless ENV['TRAVIS'] && ENV['TRAVIS']
+    next if result.nil?
+
+    identity_file = './id_ed25519'
+    unless File.exist?(identity_file)
+      puts('deploy key not found, skipping git commit & push...')
```

Comment:
if there are PRs from unprivileged users (i.e. users without write access to this repo) travis won't expose any encrypted information (i.e. env vars and files) into the build.

this prevents sensitive information from being leaked.

but if you want to allow for clean/successful builds of unprivileged PRs, you need to skip certain parts.